src/config_web.py

```diff
@@ -35,6 +36,12 @@
 # Query Customizations
 # *****************************************************************************
 
+for idx, handler in enumerate(APP_LIST):
+    if handler[0].endswith("metadata/?"):
+        handler_list = list(handler)
+        handler_list[1] = "web.handlers.metadata.MyGenesetMetadataSourceHandler"
+        handler = tuple(handler_list)
+        APP_LIST[idx] = tuple(handler_list)
```

Comment:
Or a slightly simple version:
```
APP_LIST[idx] = (handler_list[0],  "web.handlers.metadata.MyGenesetMetadataSourceHandler")
```

Let's also add a line of comment to explain what this `for` block is doing.

Other than these, looks great to me👍